CHANGELOG.md

```diff
@@ -105,6 +105,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Prime
 
+- Fixed: Crashes in several rooms when pressing start to skip cutscene exactly 1 second from the end of cutscene
+- Fixed: Incorrect shield texture for vertical **Power Beam Only** doors
+- Fixed: Elite Research - Stutter when loading room
+- Fixed: Ruined Shrine Access - Still playing Beetle fight music after leaving
+- Fixed: Research Entrance - Soflock if player kills pirates before touching the cutscene trigger (reverse labs)
+- Fixed: Mine Security Station - Wave Pirates not dropping down if the player didn't skip the cutscene immediately
+- Fixed: Uncrashed Frigate - Escape music sometimes playing at inappropriate times
+- Fixed: Furnace - Ghost elements re-appearing when re-entering from East Furnace Access
+- Fixed: Ruined Courtyard - Water platforms breaking when skipping cutscene on the last possible frame
```

Comment:
Line 116
This one should not be included as it's a fix of a new feature that never got released, doesn't make much sense to me to put it in.